pkg/controller/ec2/routetable/controller.go

```diff
@@ -237,33 +250,110 @@ func (e *external) Delete(ctx context.Context, mgd resource.Managed) error {
 	return awsclient.Wrap(resource.Ignore(ec2.IsRouteTableNotFoundErr, err), errDelete)
 }
 
-func (e *external) createRoutes(ctx context.Context, tableID string, desired []v1alpha4.Route, observed []v1alpha4.RouteState) error {
+func (e *external) deleteRoutes(ctx context.Context, tableID string, desired []v1alpha5.Route, observed []v1alpha5.RouteState) error { // nolint:gocyclo
```

Comment:
```suggestion
func (e *external) deleteRoutes(ctx context.Context, cr *v1alpha5.Route) error { // nolint:gocyclo
```

Seems like this would be enough for `deleteRoutes` to work.